src/sdsc/__init__.py

```diff
@@ -32,50 +32,18 @@
 from lxml import etree
 from .cli import printcolor, parseargs
 from .version import __version__
+from .regex import (parentheses, sentenceends, lastsentenceends,
```

Comment:
Then again, the regexes are here, they work and they might remain there for some more time. Please just import the whole module for the moment.
(Unless you really want to use your vacation to convert everything to use SpaCy or nltk and make sure there are no huge regressions.)